base/dict.jl

```diff
@@ -776,20 +776,60 @@ struct ImmutableDict{K,V} <: AbstractDict{K,V}
 end
 
 """
-    ImmutableDict
+    ImmutableDict(key=>value, key=>value, ...)
```

Comment:
It's maybe confusing to reuse the same key and value names, I'd instead write it as 

```suggestion
    ImmutableDict(key1=>value1, key2=>value2, ...)
```
or something similar